src/AppBundle/Twig/Extension/AppExtension.php

```diff
@@ -4,19 +4,13 @@
 
 use Twig\Environment;
 use Twig\Extension\AbstractExtension;
-use Twig\TwigFilter;
-use Twig\TwigFunction;
 
 class AppExtension extends AbstractExtension
 {
     public function __construct(Environment $environment)
     {
-        $environment->registerUndefinedFunctionCallback(function ($name) {
-            return new TwigFunction($name, [$this, 'doNothing']);
-        });
-        $environment->registerUndefinedFilterCallback(function ($name) {
-            return new TwigFilter($name, [$this, 'doNothing']);
-        });
+        $environment->registerUndefinedFunctionCallback(function () {});
```

Comment:
I'm impressed if one of you knows can tell me why we do this.
But if you knows, don't be like me : document it 🙂